src/main.rs

```diff
@@ -128,8 +128,7 @@ fn parse_args() -> RaftCliArgs {
     let log_level = match matches.occurrences_of("verbose") {
         0 => log::LevelFilter::Warn,
         1 => log::LevelFilter::Info,
-        2 => log::LevelFilter::Debug,
```

Comment:
This shouldn't be removed, but an evaluation of what messages are being logged as "debug" and what are logged as "trace".